src/Commands/Sql/SqlDatabaseListCommand.cs

```diff
@@ -0,0 +1,85 @@
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Sql;
+using AzureMcp.Models.Command;
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace AzureMcp.Commands.Sql;
+
+public sealed class SqlDatabaseListCommand() : BaseCommand
+{
+    private static readonly Option<string> SubscriptionOption = new Option<string>(
```

Comment:
These would be first declared in AzureMcp.Models.Argument.ArgumentDefinitions and the ToOption() extension method is used.

Reply:
Fixed